uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/BrokerCompanyPortal.java

```diff
@@ -258,6 +281,12 @@ public void enrollDevice(@NonNull final String username,
         enrollmentPerformedSuccessfully = true;
     }
 
+    @Override
+    public void enrollDevice(@NonNull final String username,
+                             @NonNull final String password) {
+        enrollDevice(username, password, false);
```

Comment:
The reason for making this change is because the flow for federated user to enroll using CP is different. the resourceIds of username and pwd fields are different.